commerce-add-ons/payment-providers/stripe/how-to-guides/testing-stripe-webhooks-locally.md

```diff
@@ -13,20 +13,47 @@ You could expose your website through your network's firewall or use tools that
 
 The following guide will use [ngrok](https://ngrok.com/) to create temporary tunnels through your network.
 
-## Step 1: Install ngrok
+## Using Stripe CLI
+### Step 1: Install the Stripe CLI and Login into the CLI
+
+Head over to [stripe.com/docs/stripe-cli](https://stripe.com/docs/stripe-cli) and follow steps 1 and 2 to install the CLI and to log in.
+
+### Step 2: Forward the stripe events to your local environment.
+
+Whilst running the site locally, make a note of your local store domain, e.g. `https:localhost:44321`. Using the Stripe CLI, to can configure Stripe to forward any events to that url. 
+
+To do so, run the following from the command line.
+
+```
+stripe listen --forward-to {local_store_domain}/umbraco/commerce/payment/callback/stripe-checkout/{payment_method_id}/
+```
+
+The `{payment_method_id}` is configured as part of the Stripe [webhook configuration](../configuring-stripe.md#step-3-webhook) step.
+
+e.g. 
+```
+stripe listen --forward-to https:localhost:44321/umbraco/commerce/payment/callback/stripe-checkout/7fb00000-0000-0000-0000-000019094a7a/
```

Comment:
should be `https://`

Reply:
Should add a reminder about go to backoffice > Stripe payment method setting and updating webhook signing secret 
![image](https://github.com/umbraco/UmbracoDocs/assets/145000730/d9efdb3e-152b-44f4-864c-9f71848701fa)

Reply:
@eshanrnh: The rest looks good to me.